<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAuSectionRdfEditWidget</class>
 <widget class="QWidget" name="CAuSectionRdfEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>319</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="badgeLabel">
         <property name="text">
          <string>Badge:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KLineEdit" name="badgeLineEdit">
         <property name="clickMessage">
          <string>Badge is shown in outliner near section title</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>Title:</string>
         </property>
         <property name="buddy">
          <cstring>titleLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KLineEdit" name="titleLineEdit">
         <property name="contextMenuEnabled">
          <bool>false</bool>
         </property>
         <property name="urlDropsEnabled">
          <bool>false</bool>
         </property>
         <property name="clickMessage">
          <string>Enter title here</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="statusLabel">
         <property name="text">
          <string>Status:</string>
         </property>
         <property name="buddy">
          <cstring>statusComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="statusComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="synopLabel">
       <property name="text">
        <string>Section description:</string>
       </property>
       <property name="buddy">
        <cstring>synopRichTextWidget</cstring>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="undoToolButton">
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="redoToolButton">
         <property name="text">
          <string>Redo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="boldToolButton">
         <property name="text">
          <string>Bold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="italicToolButton">
         <property name="text">
          <string>Italic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="underlineToolButton">
         <property name="text">
          <string>Underline</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="leftAlignToolButton">
         <property name="text">
          <string>LeftAlign</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="centerAlignToolButton">
         <property name="text">
          <string>CenterAlign</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="rightAlignToolButton">
         <property name="text">
          <string>RightAlign</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="KRichTextWidget" name="synopRichTextWidget">
       <property name="clickMessage">
        <string>Enter synopsis here</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KRichTextEdit</class>
   <extends>KTextEdit</extends>
   <header>krichtextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KRichTextWidget</class>
   <extends>KRichTextEdit</extends>
   <header>krichtextwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
